controls/radautocompletebox/styles-and-templates/customizing-drop-down-itemtemplate.md

```diff
@@ -1,18 +1,16 @@
 ---
-title: Customizing the DropDownItemTemplate
-page_title: Customizing the DropDownItemTemplate
-description: Customizing the DropDownItemTemplate
+title: Customizing the DropDownItemTemplate and DropDownItemTemplateSelector
+page_title: Customizing the DropDownItemTemplate and DropDownItemTemplateSelector
+description: The RadAutoCompleteBox control expose DropDownItemTemplate and DropDownItemTemplateSelector which can be used to customize the drop down item.

```

Comment:
.. control **exposes** **the** DropDownItemTemplate and DropDownItemTemplateSelector **properties** which can ..